package.json

```diff
@@ -18,20 +18,22 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "files": [
-    "dist/**/*"
+    "dist/**/*.map",
+    "dist/**/*.js",
+    "dist/**/*.d.ts"
   ],
   "engines": {
     "node": ">=12.13.0",
     "npm": ">=6.12.0"
   },
   "scripts": {
     "prepare": "npm run build",
-    "build": "tsc",
+    "build": "npm run build:clean && tsc --build src",
```

Comment:
More information on `tsc --build` available in the [Project References](https://www.typescriptlang.org/docs/handbook/project-references.html) TypeScript documentation.